calc.js

```diff
@@ -1,294 +1,85 @@
 window.onload = function(){
+    
+    var calculator = document.getElementById("calculator");
+    var keys = calculator.querySelector(".keys");
+    keys.addEventListener('click', function(e){
+        if(e.target.attributes.type.value === "numeral"){
+            digitPressed(e.target.innerHTML);              //this needs conditions for nought and point //
+            
+        }else if(e.target.attributes.type.value === "operator"){
+            operatorPressed(e.target.innerHTML);
+            e.target.classList.add ("pressed");
+            
+        }else if(e.target.attributes.type.value === "negative"){
```

Comment:
Why does subtraction have its own type?

Reply:
subtraction functions differently than the other operators. I want to stop things like "3 / * 8" which would break the calculator but "3 / - 8" is legit.